tests/contract_testcases/substrate/tags/inheritdoc.sol

```diff
@@ -0,0 +1,18 @@
+
+contract x {}
+
+contract a {}
+
+contract b is a {}
+
+contract c is b {
+    /// @inheritdoc a
+    int v1;
+
+    /// @inheritdoc x
+    int v2;
+
+}
+
+// ---- Expect: diagnostics ----
+// error: 12:21-22: base contract 'x' not found in tag '@inheritdoc'
```

Comment:
This read negative symbol in the file means we are not adding a new line at the end of it. GitHub complains about this all the time. Should we adopt the standard of adding a new line at the end of all files?

Reply:
It's not really a warning. It's an indicator that there is no newline, and it's displayed because[ the diff would look the same](https://stackoverflow.com/a/5813359) regardless whether the file has a newline at the end or not